examples/lock-app/efr32/README.md

```diff
@@ -55,20 +56,7 @@ example, the output binary will be lacking key features (e.g. OpenThread).
 -   Install some additional tools(likely already present for CHIP developers):
 
            # Linux
-           $ sudo apt-get install git make automake libtool ccache libwebkitgtk-1.0-0 ninja
-
-           # Mac OS X
-           $ brew install automake libtool ccache ninja
-
--   To build for an MG21 part make the following changes to the
-    platform/CMSIS/Include/core_cm33.h file within the Silicon Labs SDK. Copy
-    the following lines to the top of the core_cm33.h file.
-
-```cpp
-#if defined(__GNUC__)
-#pragma GCC diagnostic ignored "-Wpedantic"
-#endif
```

Comment:
This part is no longer relevant?

Reply:
Since we don't compile with -Wpedantic, and because we are quite far from being able to compile with that option even if we wanted to, it doesn't seem relevant. I think it was copy/pasted from somewhere.